docker/entrypoint.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
```

Comment:
We might want to move this under a directory like `keras-to-tensorflow` in here so that the image can be distributed as `triage/keras-to-tensorflow`, then we can do similar with `keras-to-coreml` and any future converters that we want to be encapsulated like this.